#global testerror "Err0102"
func toto() throw
{
    throw A{}
}
